sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -297,6 +305,17 @@ public KalixTestKit start(final Config config) {
     return this;
   }
 
+  private int getEventingBackendPort(Settings.EventingSupport eventingSupport) {
+    switch(eventingSupport) {
+      case KAFKA:
+        return DEFAULT_KAFKA_PORT;
```

Comment:
So we actually only support it on the default port?

Reply:
I do think being able to provide a different port would be good. But we don't support for pubsub either, so I thought about opening a follow-up issue and allow a different port for both.

Reply:
Let's leave it as is. I was thinking about the scenario where a user wants to connect to a cloud Kafka, where they often use some arbitrary port, but we can add support the day a user actually wants to do that.